tests/unit/amo/reducers/testLanding.js

```diff
@@ -53,6 +53,30 @@ describe('landing reducer', () => {
 
       expect(state.resultsLoaded).toEqual(false);
     });
+
+    it('resets each set of add-ons', () => {
+      const entities = {
+        addons: {
+          bar: { slug: 'bar' },
```

Comment:
I think it would be best to use actual add-ons here rather than the stubs. It should be as easy as using `import { fakeAddon } from 'tests/unit/amo/helpers';` and doing: `bar: { ...fakeAddon, slug: 'bar' },`.

Just in case the reducers start to rely more on the structure of the add-on.